functions/Read-DbaBackupHeader.ps1

```diff
@@ -0,0 +1,153 @@
+﻿Function Read-DbaBackupHeader
+{
+<#
+.SYNOPSIS 
+Reads and displays detailed information about a SQL Server backup
+
+.DESCRIPTION
+Reads full, differential and transaction log backups. An online SQL Server is required to parse the backup files and the path specified must be relative to that SQL Server.
+	
+.PARAMETER SqlServer
+The SQL Server instance. 
+
+.PARAMETER SqlCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. 
+
+.PARAMETER Path
+Path to SQL Serer backup file. This can be a full, differential or log backup file.
+	
+.PARAMETER Simple
+Returns fewer columns for an easy overview
+	
+.PARAMETER FileList
+Returns detailed information about the files within the backup	
+
+.NOTES 
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
```

Comment:
Then it makes it look weird in Get-Help. Allowing PS to word wrap instead of us (because people likely wouldn't stop at 80chars) ensures it'll be pretty when using Get-Help